tests/lib/rules/max-lines-per-function.js

```diff
@@ -0,0 +1,344 @@
+/**
+ * @fileoverview Tests for max-lines-per-function rule.
+ * @author Pete Ward <peteward44@gmail.com>
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+const rule = require("../../../lib/rules/max-lines-per-function");
+const RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const parserOptions = { ecmaVersion: 6 };
+
+const ruleTester = new RuleTester();
```

Comment:
Optionally, you could pass in the parserOptions to the RuleTester constructor since (I assume) no test actually needs to run with ecmaVersion: 5. This will result in all tests running with ecmaVersion: 6 by default.